index.html

```diff
@@ -85,47 +83,177 @@ <h2 class="second-header">I am software developer</h2>
+                <li><h3>JavaScript</h3></li>
+                <li><h3>html</h3></li>
+              </ul>
+              <button type="button" class="see-project">See Project</button>
+            </div>
+          </div>
+          <div class="card-work">
+            <div class="img-placeholder"></div>
+            <div class="card-details">
+              <div class="card-description">
+                <h2>Multi-Post Stories Gain+Glory</h2>
+              </div>
+              <ul class="languages">
+                <li><h3>Ruby on rails</h3></li>
+                <li><h3>css</h3></li>
+                <li><h3>JavaScript</h3></li>
+                <li><h3>html</h3></li>
+              </ul>
+              <button type="button" class="see-project">See Project</button>
+            </div>
           </div>
         </div>
-      </div>
+      </section>
+      <section class="about-me">
+        <div class="about-me-description">
+          <h2>About me</h2>
+          <p class="intro-text">
+            I can help you build a product , feature or website Look through
+            some of my work and experience! If you like what you see and have a
+            project you need coded, don’t hestiate to contact me.
+          </p>
+          <button type="button" class="get-resume">get my Resume</button>
+        </div>
 
-      <div class="get-in-touch">
-        <p class="intro-to-form"></p>
-        <!-- <form type action="">
+        <ul class="card-container">
+          <li class="card-about">
+            <div class="img-placeholder">
+              <img
+                src="./assets/card-images/icon-languages.svg"
+                alt="Icon image for programming languages i know"
+              />
+            </div>
+            <div class="card-details">
+              <h2>languages</h2>
+              <ul class="languages">
+                <li><h3>JavaScript</h3></li>
+                <li><h3>Ruby</h3></li>
+                <li><h3>Html</h3></li>
+                <li><h3>CSS</h3></li>
+              </ul>
+            </div>
+          </li>
+          <li class="card-about">
+            <div class="img-placeholder">
+              <img
+                src="./assets/card-images/icon-frameworks.svg"
+                alt="Icon image for programming languages i know"
+              />
+            </div>
+            <div class="card-details">
+              <h2>Frameworks</h2>
+              <ul class="languages">
+                <li><h3>Bootstrap</h3></li>
+                <li><h3>Ruby on Rails</h3></li>
+                <li><h3>Rspec</h3></li>
+                <li><h3>Capybara</h3></li>
+                <li><h3>Selenium</h3></li>
+              </ul>
+            </div>
+          </li>
+          <li class="card-about">
+            <div class="img-placeholder">
+              <img
+                src="./assets/card-images/icon-skills.svg"
+                alt="Icon image for programming languages i know"
```

Comment:
- [ Optional]  I think it would be better if you made your website responsive also for pages with 320 px i.e to make it match the design exactly at that width
your version  
![image](https://user-images.githubusercontent.com/74744787/159111180-3a1e3b75-4651-4b92-9b59-8b7b20cf231f.png)

Figma version 
![image](https://user-images.githubusercontent.com/74744787/159111267-7ae1d9ce-38ae-451f-bd65-af5751d3fcd4.png)